data_and_models/pipelines/ner/transformers_vs_spacy/transformers/1_run_transformers_ner.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# This script is from huggingface/transformers v4.4.0.
+# It is from https://github.com/huggingface/transformers/tree/v4.4.0/examples/token-classification/run.sh.
+
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+exp_name="evaluate_transformers"
+
+#  --model_name_or_path bert-base-uncased \
+#  --lr_scheduler_type "constant" \
+DS="train_data.pkl"
+DS_VAL="test_data.pkl"
+python3 1_run_transformers_ner.py \
+  --model_name_or_path "dmis-lab/biobert-large-cased-v1.1" \
+  --output_dir "checkpoints/$exp_name" \
+  --do_train \
+  --do_eval \
+  --do_predict \
+  --evaluation_strategy "steps" \
+  --eval_steps 10 \
+  --train_file "$DS" \
+  --validation_file "$DS_VAL" \
```

Comment:
Is it expected that the `test` data are used as `dev` data in this PR?

This isn't what we are doing at the moment.

The reported performances are therefore not comparable.

Reply:
Here the `$DS_VAL` is used only for the evaluation of the trained model, but no hyperparameter is chosen based on the score of the model on `$DS_VAL` (this is different from `spacy` that instead chooses the optimal `n_epoch` based on the best score achieved on the `valid_split`).

So effectively this `test` data is treated just as `test` data, not as a `valid` split, and scores are comparable.